frontend/js/components/template/Card.jsx

```diff
@@ -128,7 +128,7 @@ class Card extends React.Component {
         if (this.props.card.loadingCardTemplateError) {
             return this.renderLoadTemplateException();
         }
-        const mlsButtonDisable = (!includes(window.location.hash, 'map') && isEmpty(this.props?.rowData?.geometry?.coordinates)) || isEmpty(this.props.geometryType);
+        const mlsButtonDisable = isEmpty(this.props?.rowData?.geometry?.coordinates);
```

Comment:
Instead of disabling the button, maybe it is button to now show it at all, what do you think?

Reply:
Yes, makes sense. I will modify it accordingly